app-constants.js

```diff
@@ -23,16 +23,13 @@ const kEnvironmentVariables = [
   "AWS_SECRET_ACCESS_KEY",
 ];
 
-const AppConstants = {
-  init() {
-    for (const v of kEnvironmentVariables) {
-      if (process.env[v] === undefined) {
-        throw new Error(`Required environment variable was not set: ${v}`);
-      }
-      this[v] = process.env[v];
-    }
-    return this;
-  },
-};
+const AppConstants = { };
```

Comment:
Should this be a `Map()` instead, and/or, should we use [`Object.freeze()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/freeze) (when exporting) to make the object immutable-_ish_?

Reply:
Filed #58